cmd/ci-operator/main.go

```diff
@@ -2270,17 +2270,25 @@ func (o *options) getResolverInfo(jobSpec *api.JobSpec) *api.Metadata {
 	}
 
 	// identify org, repo, and branch from refs object
+	refsTotal := 0
 	for _, ref := range allRefs {
 		if ref.Org != "" && ref.Repo != "" && ref.BaseRef != "" {
 			info.Org += fmt.Sprintf("%s,", ref.Org)
 			info.Repo += fmt.Sprintf("%s,", ref.Repo)
 			info.Branch += fmt.Sprintf("%s,", ref.BaseRef)
+			refsTotal++
 		}
 	}
```

Comment:
NIT: feel like this is simpler.

```suggestion
	for i, ref := range allRefs {
		if ref.Org != "" && ref.Repo != "" && ref.BaseRef != "" {
			info.Org += fmt.Sprintf("%s,", ref.Org)
			info.Repo += fmt.Sprintf("%s,", ref.Repo)
			info.Branch += fmt.Sprintf("%s,", ref.BaseRef)
			if info.Variant != "" && i > 0 {
				info.Variant = fmt.Sprintf("%s,", info.Variant)
			}
		}
	}
```